pandas/tests/tools/test_to_numeric.py

```diff
@@ -912,7 +912,6 @@ def test_to_numeric_use_nullable_dtypes_already_nullable(dtype):
 @pytest.mark.parametrize(
     "use_nullable_dtypes, dtype", [(True, "Float64"), (False, "float64")]
 )
-@pytest.mark.parametrize("dtype_backend", ["pandas", "pyarrow"])
 def test_to_numeric_use_nullable_dtypes_error(
```

Comment:
I think the `pytest.importorskip("pyarrow")` can be removed within the test now

Reply:
Ah good point, thx